File: app/src/main/java/hibernate/v2/testyourandroid/model/ExtendedBluetoothDevice.java
package hibernate.v2.testyourandroid.model;

/**
 * Created by himphen on 24/5/16.
 */

public class ExtendedBluetoothDevice {

	private String name;
	private String riss;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getRiss() {
		return riss;
	}

	public void setRiss(int riss) {
		this.riss = "-" + riss + " dBm";
	}
}

File: app/src/main/java/hibernate/v2/testyourandroid/model/ExtendedBluetoothDevice.kt
package hibernate.v2.testyourandroid.model

/**
 * Created by himphen on 24/5/16.
 */
data class ExtendedBluetoothDevice(var name: String, var rssi: Int) {
    fun getRssi(): String {
        return "-$rssi dBm"
    }
}
